### PR TITLE
Fix injection of ZopeLite test configuration into the global Zope configuration.

### PR DESCRIPTION
**Pull request by thet**
Fixese #64

### CODE REVIEW

**Comment by thet** [comment]
@jenkins-plone-org please run jobs

**Comment by thet** [comment]
@jenkins-plone-org please run jobs

**Comment by pbauer** [comment]
@thet Merging this broke all of our jenkins tests like this:

```
Running plone.app.theming.testing.Theming:Functional tests:
  Set up plone.testing.zope.Startup Traceback (most recent call last):
  File "/home/jenkins/.buildout/eggs/cp27m/zope.testrunner-5.0-py2.7.egg/zope/testrunner/runner.py", line 451, in run_layer
    setup_layer(options, layer, setup_layers)
  File "/home/jenkins/.buildout/eggs/cp27m/zope.testrunner-5.0-py2.7.egg/zope/testrunner/runner.py", line 787, in setup_layer
    setup_layer(options, base, setup_layers)
  File "/home/jenkins/.buildout/eggs/cp27m/zope.testrunner-5.0-py2.7.egg/zope/testrunner/runner.py", line 787, in setup_layer
    setup_layer(options, base, setup_layers)
  File "/home/jenkins/.buildout/eggs/cp27m/zope.testrunner-5.0-py2.7.egg/zope/testrunner/runner.py", line 787, in setup_layer
    setup_layer(options, base, setup_layers)
  File "/home/jenkins/.buildout/eggs/cp27m/zope.testrunner-5.0-py2.7.egg/zope/testrunner/runner.py", line 787, in setup_layer
    setup_layer(options, base, setup_layers)
  File "/home/jenkins/.buildout/eggs/cp27m/zope.testrunner-5.0-py2.7.egg/zope/testrunner/runner.py", line 787, in setup_layer
    setup_layer(options, base, setup_layers)
  File "/home/jenkins/.buildout/eggs/cp27m/zope.testrunner-5.0-py2.7.egg/zope/testrunner/runner.py", line 792, in setup_layer
    layer.setUp()
  File "/home/jenkins/workspace/pull-request-5.2/src/plone.testing/src/plone/testing/zope.py", line 311, in setUp
    self.setUpDatabase()
  File "/home/jenkins/workspace/pull-request-5.2/src/plone.testing/src/plone/testing/zope.py", line 469, in setUpDatabase
    raise Exception('You try to run plone.testing tests together with '
Exception: You try to run plone.testing tests together with ZopeTestCase tests. This will result in random failures. Convert the ZopeTestCase Tests or do not run them together

```

You can easily reproduce it locally like this: `./bin/test -s Products.DCWorkflow -s plone.namedfile -D`

I don't really know what the warning is good for. We have not had random errors since we fixed all test-isolation-issues.

**Comment by datakurre** [comment]
🙈

File: src/plone/testing/zope.py [hunk 1]
```diff
@@ -461,6 +460,11 @@ def setUpDatabase(self):
         that the database that is opened by Zope 2 is in fact the top of
         the resource stack.
         """
+
+        # NOTE: Keep that import here.
+        #       Having it on module-level has the side effect of injecting the
+        #       ZopeLite test configuration into the global Zope configuration.
+        from Testing.ZopeTestCase.ZopeLite import _patched as ZOPETESTCASEALERT
```
Comment (thet) [inline src/plone/testing/zope.py]: I think we can remove this and the following lines all together?
Looking at it, I can't make sense of importing something and then throwing an error if the import was successful.

Comment (pbauer) [reply src/plone/testing/zope.py]: I agree. It seems that running `ZopeTestCase` together with ours had side-effects that needed to be warned about. See https://github.com/plone/plone.testing/commit/3268cda0ecddab51dbe7bfc3c7c7b997777f9ac0 
I should hope that these issues are gone.

Comment (thet) [reply src/plone/testing/zope.py]: Ah, yes, that makes sense.
I remeber also, that was done at the Alpine City Sprint.

**Pull request merged**